spec/services/fast_track/hypertension_observation_data_spec.rb

```diff
@@ -9,128 +9,35 @@
 
   let(:response) do
     # Using specific test ICN below:
-    client = Lighthouse::VeteransHealth::Client.new(2_000_163)
+    client = Lighthouse::VeteransHealth::Client.new(32_000_225)
     client.get_resource('observations')
   end
 
+  let(:response_with_recent_bp) do
+    original_first_bp_reading = response.body['entry'].first
+    original_first_bp_reading['resource']['issued'] = (Time.zone.today - 2.weeks).to_s
+    response
+  end
+
   describe '#transform' do
-    it 'returns the expected hash' do
-      expect(described_class.new(response).transform)
+    it 'returns only bp readings from the past year', :vcr do
+      expect(described_class.new(response_with_recent_bp).transform)
```

Comment:
I'm seeing a spec failure on this line when I run the specs locally:

```
Failures:

  1) FastTrack::HypertensionObservationData#transform returns only bp readings from the past year
     Failure/Error: 
       expect(described_class.new(response_with_recent_bp).transform)
         .to match(
           [
             {
               issued: '2021-12-14',
               practitioner: 'DR. THOMAS359 REYNOLDS206 PHD',
               organization: 'LYONS VA MEDICAL CENTER',
               systolic: {
                 'code' => '8480-6',
                 'display' => 'Systolic blood pressure',

       Expected [{ issued: "2021-12-16", practitioner: "DR. THOMAS359 REYNOLDS206 PHD", organization: "LYONS VA MEDICAL CENTER", systolic: { "code" => "8480-6", "display" => "Systolic blood pressure", "unit" => "mm[Hg]", "value" => 175.0 }, diastolic: { "code" => "8462-4", "display" => "Diastolic blood pressure", "unit" => "mm[Hg]", "value" => 111.0 } }]
       to match [{ issued: "2021-12-14", practitioner: "DR. THOMAS359 REYNOLDS206 PHD", organization: "LYONS VA MEDICAL CENTER", systolic: { "code" => "8480-6", "display" => "Systolic blood pressure", "value" => 175.0, "unit" => "mm[Hg]" }, diastolic: { "code" => "8462-4", "display" => "Diastolic blood pressure", "value" => 111.0, "unit" => "mm[Hg]" } }]

       Diff:

       ┌ (Key) ──────────────────────────┐
       │ ‹-› in expected, not in actual  │
       │ ‹+› in actual, not in expected  │
       │ ‹ › in both expected and actual │
       └─────────────────────────────────┘

         [
           {
       -     issued: "2021-12-14",
       +     issued: "2021-12-16",
             practitioner: "DR. THOMAS359 REYNOLDS206 PHD",
             organization: "LYONS VA MEDICAL CENTER",
             systolic: {
               "code" => "8480-6",
               "display" => "Systolic blood pressure",
               "unit" => "mm[Hg]",
               "value" => 175.0
             },
             diastolic: {
               "code" => "8462-4",
               "display" => "Diastolic blood pressure",
               "unit" => "mm[Hg]",
               "value" => 111.0
             }
           }
         ]
     # ./spec/services/fast_track/hypertension_observation_data_spec.rb:24:in `block (3 levels) in <top (required)>'
```

Reply:
Thanks - fixed in b7b43f2